src/utils/xmlPartsUtils.ts

```diff
@@ -27,15 +28,19 @@ const updateWorkbookDataAndConfigurations = async (zip: JSZip, fileConfigs?: Fil
     await tableUtils.updateTableInitialDataIfNeeded(zip, tableData, updateQueryTable);
 };
 
-const updateWorkbookPowerQueryDocument = async (zip: JSZip, queryName: string, queryMashupDoc: string): Promise<void> => {
+const updateWorkbookPowerQueryDocument = async (zip: JSZip, queryName: string, queryMashupDoc: string, connectionOnlyQueryNames?: string[]): Promise<void> => {
     const old_base64: string | undefined = await pqUtils.getBase64(zip);
 
     if (!old_base64) {
         throw new Error(base64NotFoundErr);
     }
 
     const new_base64: string = await replaceSingleQuery(old_base64, queryName, queryMashupDoc);
```

Comment:
please add few comments about the flow of add queries... first replacing the exiting loaded query name... second ....

Reply:
Will be helpful for who not familier the code, we are starting to have a different paths based on the arguments it become confusing